app/models/solidus_subscriptions/user_generator.rb

```diff
@@ -32,6 +34,9 @@ def create_stub_user(order)
         password: initial_password,
         password_confirmation: initial_password,
       }
+      if Spree.user_class.new.respond_to?(:team_id)
```

Comment:
Seeing that we are doing this twice, should this be a method on the class? I'm guessing if we need to do it twice we might need to do it a bunch.

Also, might be misunderstanding the reasoning here.

Reply:
I don't think this will see heavy use and I would prefer to keep the dirt (I don't consider this a great solution) in one place for now.

Reply:
If it's in a decorator in this fork, we can do something like this:
```ruby
# TODO THIS IS GARBAGE AND WE WILL FIX IT LATER
# DON'T USE IT OR YOUR PR WILL BE CLOSED AND 
# YOUR DRIVE WILL BE FORMATTED
def BAD_HACK_respond_to_team?
  new.respond_to?(:team_id)
end
```
I don't love doing that (React does this a bit), but I'll tell you now that in like two months someone like me is going to copy and paste that line if we don't document that it's a short term measure.

Reply:
I added a method with a comment, a little bit like yours, see what you think.